www_src/pages/project/project.jsx

```diff
@@ -210,11 +210,15 @@ var Project = React.createClass({
       selectedEl: el.id
     });
   },
-  zoomToPage: function (coords) {
+  zoomToPage: function (coords, play) {
+    if (typeof play === 'undefined') {
+      play = true;
+    }
+
     this.setState({
       camera: this.cartesian.getFocusTransform(coords, 1),
       zoom: 1,
-      isPageZoomed: true,
+      isPageZoomed: play,
```

Comment:
This is kind of weird...should always be `true` given the method's purpose. There's already `zoomFromPage` to zoom out.